.github/workflows/docs.yml

```diff
@@ -6,12 +6,12 @@ on:
       - main
     tags:
       - 'v*'
-  pull_request:
-    branches:
-      - main
+  pull_request_review:
+    types: [submitted]
 
 jobs:
   docs:
+    if: github.ref == 'refs/heads/main' || startsWith(github.ref, 'refs/tags/v') || (github.event_name == 'pull_request_review' && github.event.review.state == 'approved')
```

Comment:
so when merging to `main` does it still count as a `pull_request_review` event?

Reply:
nop, this condition now means
`push to main` or `pushed a tag with v*` or `a pull_request_review event and it is an approval`

Reply:
so it should trigger when satisfy any of the three

Reply:
ah right! nice, seems to have worked :)